src/homework5/get_ranges.py

```diff
@@ -0,0 +1,27 @@
+"""Function get_ranges.
+
+Реализовать функцию get_ranges которая получает на вход непустой список
+неповторяющихся целых чисел, отсортированных по возрастанию, которая этот
+список “сворачивает”
+get_ranges([0, 1, 2, 3, 4, 7, 8, 10]) // "0-4,7-8,10"
+get_ranges([4,7,10]) // "4,7,10"
+get_ranges([2, 3, 8, 9]) // "2-3,8-9
+"""
+
+
+def get_ranges(lst):
+    """Compress list of integers into a string."""
+    import itertools
+
+    res = []
+    for key, group in itertools.groupby(enumerate(lst), lambda x: x[1] - x[0]):
+        group = list(group)
+        res.append((group[0][1], group[-1][1]))
```

Comment:
magic numbers

Reply:
Right, but unfortunately I have no idea how to get rid of these magic numbers.

Reply:
Explicit is better than implicit. that means it can be not the best idea for solution but it's ok that you know the standard library